onnxruntime/test/mlas/unittest.cpp

```diff
@@ -1240,11 +1622,13 @@ main(
     void
     )
 {
-//    ExecuteSgemmTests();
-    ExecuteConvTests();
-//    ExecutePool2DTests();
+//    MlasSgemmTest::Execute();
+//    MlasConv2DTest::Execute();
+    MlasNchwcConv2DTest::Execute();
+//    MlasPool2DTest::Execute();
+//    MlasNchwcPool2DTest::Execute();
+//    MlasPool3DTest::Execute();
 //    ExecutePool3DTests();
```

Comment:
Are these tests run as part of the CI build?

What's the current code coverage from these tests?

Are the commented out tests intentionally disabled, and if so when would they be re-enabled?